README.md

```diff
@@ -174,6 +174,7 @@ If you want to test it locally, see [Docker](#docker).
 | `endpoints[].client`                            | [Client configuration](#client-configuration).                                                                                                     | `{}`                       |
 | `endpoints[].ui`                                | UI configuration at the endpoint level.                                                                                                            | `{}`                       |
 | `endpoints[].ui.hide-hostname`                  | Whether to include the hostname in the result.                                                                                                     | `false`                    |
+| `endpoints[].ui.hide-url`                       | Whether to include the URL in the result.                                                                                                     | `false`                    |
```

Comment:
```suggestion
| `endpoints[].ui.hide-hostname`                  | Whether to hide the hostname in the result.                                                                                                        | `false`                    |
| `endpoints[].ui.hide-url`                       | Whether to ensure the URL is not displayed in the results. Useful if the URL contains a token.  | `false`                    |
```
(If you could also align it, I'd appreciate it 🙏 GitHub doesn't make aligning tables easy during code reviews)